tests/snappi_tests/multidut/pfc/files/lossless_response_to_external_pause_storms_helper.py

```diff
@@ -316,7 +317,12 @@ def __gen_data_flow(testbed_config,
         flow = testbed_config.flows.flow(name='{} {} -> {}'.format(flow_name_prefix, src_port_id, dst_port_id))[-1]
         flow.tx_rx.port.tx_name = testbed_config.ports[src_port_id].name
         flow.tx_rx.port.rx_name = testbed_config.ports[dst_port_id].name
-        eth, ipv4 = flow.packet.ethernet().ipv4()
+        eth, ipv4, udp = flow.packet.ethernet().ipv4().udp()
+        src_port = random.randint(5000, 6000)
```

Comment:
Do we have any common RDMA traffic pattern to reference with? TCP or UDP? and source port, destination port range? if we have, we can try to make it closer to real traffic.